tests/unit/lms/services/user_test.py

```diff
@@ -218,8 +274,11 @@ def assignment(self, course):
         return assignment
 
     @pytest.fixture
-    def student_in_assigment(self, assignment, application_instance):
+    def student_in_assignment(self, assignment, application_instance, db_session):
```

Comment:
Adapting the existing fixtures to the new tables while keeping the old ones.

This reflects what happens in productions and should allow us to transition to the new ones only deleting the older references.